package-scripts.js

```diff
@@ -82,27 +82,19 @@ module.exports = {
                 ),
                 pretest: 'tslint -c ./tslint.json -t stylish ./test/integration/**/*.ts',
                 verbose: 'nps "test.integration --verbose"',
+                // -i. Run all tests serially in the current process, rather than creating a worker pool of child processes that run tests. This can be useful for debugging.
                 run: 'cross-env NODE_ENV=test jest --testPathPattern=integration -i',
             },
             e2e: {
-                default: {
-                    script: series(
-                        'nps banner.test',
-                        'nps test.e2e.pretest',
-                        runInNewWindow(series('nps build', 'nps start')),
-                        'nps test.e2e.run'
-                    )
-                },
-                pretest: {
-                    script: 'tslint -c ./tslint.json -t stylish ./test/e2e/**/*.ts'
-                },
-                verbose: {
-                    script: 'nps "test.e2e --verbose"'
-                },
-                run: series(
-                    `wait-on --timeout 120000 http-get://localhost:3000/api/info`,
-                    'cross-env NODE_ENV=test jest --testPathPattern=e2e -i'
-                )
+                default: series(
+                    'nps banner.test',
+                    'nps test.e2e.pretest',
+                    'nps test.e2e.run'
+                ),
+                pretest: 'tslint -c ./tslint.json -t stylish ./test/e2e/**/*.ts',
```

Comment:
Can we refactor this command to be more generic and pass a parameter `./test/e2e/**/*.ts` so we can centralise the tslint logic.